src/twisted/runner/procmon.py

```diff
@@ -5,11 +5,50 @@
 """
 Support for starting, monitoring, and restarting child process.
 """
-from twisted.python import log
+import attr
```

Comment:
does Twisted depend on attr ?

I don't see it defined in https://github.com/twisted/twisted/blob/trunk/src/twisted/python/_setup.py#L241

I know that attr my be already there from another dependency, but we need to explicitly define it... just in case all the dependencies are moving away from it and then it will no longer be available